README.md

```diff
@@ -124,8 +125,18 @@ In order to facilitate direct comparisons between results published in the liter
 
 ```python
 datamodule = InriaAerialImageLabelingDataModule(root="...", batch_size=64, num_workers=6)
-task = SemanticSegmentationTask(segmentation_model="unet", encoder_weights="imagenet", learning_rate=0.1)
```

Comment:
Parameter names changed. At the moment, every single argument is required, although I'm planning on changing this in the future.